__tests__/app.test.js

```diff
@@ -0,0 +1,49 @@
+const request=require('supertest');
+const app = require('../app');
+const db= require('../db/connection');
+const seed = require('../db/seeds/seed');
+const data = require('../db/data/test-data');
+
+afterAll(()=>{
+    return db.end();
+})
+
+beforeEach(()=>{
+    console.log('seeding!');
+    return seed(data)
+})
+
+
+describe('GET /api/topics',()=>{
+    test ('200:responds with an array of topics', ()=>{
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({body})=>{
+            const {topics}=body;
+            console.log(topics)
+            expect(topics).toHaveLength(3);
+            topics.forEach((topic)=>{
+                expect(topic).toMatchObject({
+                    description:expect.any(String),
+                    slug: expect.any(String),
+                })
+            })
+        })
+    })
+
+
+    test.only ('404:responds with a not found message to a request to non existing endpoint', ()=>{
```

Comment:
You should really remove .onlys for PRs - in a real life scenario, this is unlikely to make it through a PR, especially when there are checks like husky involved or pipelines that check whether all tests passed before pushing up, as there is no way for the person reviewing your code whether this is the ONLY test that passed, or whether the other ones also passed :D 

This is a small thing but really bad form - using .only is really only for personal use, not for code you commit and push